README.md

```diff
@@ -144,7 +144,9 @@ $ rush react
 
 | Project                         | Description                   |
 | ------------------------------- | ----------------------------- |
-| [AI-generated Components](todo) | AI-generated chart component. |
+| [React-VChart]([todo](https://github.com/VisActor/VChart/tree/develop/packages/react-vchart)) | React interface for @VisActor/VChart |
```

Comment:
you can remove 

![Uploading image.png…]()

Reply:
It has been corrected,and cn readme also been added @kkxxkk2019